apps/project/tests/test_schemas.py

```diff
@@ -1031,6 +1031,42 @@ def test_project_role(self):
         content = self.query_check(query)
         self.assertEqual(len(content['data']['projectRoles']), project_role_count)
 
+    def test_user_pinned_projects_query(self):
+        query = '''
+            query MyQuery {
+            userPinnedProjects {
+                clientId
+                id
+                order
+            }
+        }
+        '''
+
+        user = UserFactory.create()
+        non_member_user = UserFactory.create()
```

Comment:
We aren't checking any membership logic here, let's use proper naming.

```suggestion
        user1 = UserFactory.create()
        user2 = UserFactory.create()
```